src/plone/importexport/browser/importexport.py

```diff
@@ -712,10 +712,12 @@ def getmatrix(self, headers=None, columns=4):
 
         matrix = {}
         count = len(headers)
-        rows = float(count / columns)
+        rows = float(count) / columns
 
-        if isinstance(rows, float):
```

Comment:
`rows` is always an instance of `float`. Hence this conditional statement doesn't work